zeppelin/cli.py

```diff
@@ -2,9 +2,16 @@
 import argparse
 import json
 import sys
-from .new_converter import NewConverter as nc
-from .legacy_converter import LegacyConverter as lc
+from .new_converter import NewConverter
+from .legacy_converter import LegacyConverter
 
+def check_version(text, args, directory):
+    if 'results' in text['paragraphs'][0]:
+        return NewConverter(args.in_filename, args.out_filename,
```

Comment:
This function should only check the zeppelin version, like its name says. You could even rename the function `is_new_format` since that's exactly what it does. Please leave the rest of the code in the calling function.